Header header
bool toLeft
bool engage
bool mode
float32 velocityLimit
float32 acceleration
int64 position